src/CMakeLists.txt

```diff
@@ -637,9 +637,23 @@ install(TARGETS ceph-mon DESTINATION bin)
 if(NOT ALLOCATOR STREQUAL "jemalloc")
   set(disable_jemalloc "DISABLE_JEMALLOC=1")
 endif()
+
+set(ROCKSDB_EXTRA_CXXFLAG -fPIC -Wno-unused-variable)
+if(NOT CMAKE_BUILD_TYPE MATCHES "Debug")
```

Comment:
nit, could be more consistent with existing way to check `CMAKE_BUILD_TYPE` at line:30
```cmake
if(NOT CMAKE_BUILD_TYPE STREQUAL Debug)
```